.github/workflows/firebase-hosting-merge.yml

```diff
@@ -0,0 +1,16 @@
+# This file was auto-generated by the Firebase CLI
```

Comment:
Remove the firebase workflows as this will get triggered on merge and as we do not have firebase in-place so this will always fail.